src/config/config.ts

```diff
@@ -96,6 +103,13 @@ const config: AppIds = {
       ru: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-ru',
       ar: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-ar/'
     },
+    downloadMohLandingPage: {
+      he: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-he/',
+      en: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-en/',
+      ru: 'https://govextra.gov.il/ministry-of-health/hamagen-app-ru/download-ru/',
+      ar: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-ar/',
+      am: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-he/',
```

Comment:
fix the url (if exits such url)

Reply:
I couldn't find one. please supply a link

Reply:
@kaminskypavel https://govextra.gov.il/ministry-of-health/hamagen-app-am/download-am

Reply:
updated. thanks

Reply:
This block is redundant

Reply:
Please change this url to the url used above. This is the url for the Hebrew download page, url for Amharic download page is needed here.